bind_test.go

```diff
@@ -135,7 +135,7 @@ func TestBindForm(t *testing.T) {
 	rec := httptest.NewRecorder()
 	c := e.NewContext(req, rec)
 	req.Header.Set(HeaderContentType, MIMEApplicationForm)
-	obj := []struct{ Field string }{}
+	var obj []struct{ Field string }
```

Comment:
We prefer without `var` across the project unless it is really needed, so lets stick to it.

Reply:
@vishr fixed

Reply:
<img width="520" alt="echo golang src github com labstack echo - bind_test go echo 2018-02-21 10-46-04" src="https://user-images.githubusercontent.com/888650/36468559-e0f17d6a-16f4-11e8-9b47-ac22b3865a53.png">

Reply:
I'll fix it with (just more simple):

```go
err := c.Bind(&[]struct{ Field string }{})
```